### PR TITLE
Official Release 0.32.0

### PR DESCRIPTION
**Pull request by youngbupark**
# Description

This is the official release for 0.32.0

## Type of change

- This pull request fixes a bug in Radius and has an approved issue (issue link required).
- This pull request adds or changes features of Radius and has an approved issue (issue link required).
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #issue_number

### CODE REVIEW

File: docs/release-notes/v0.32.0.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### WIP
```
Comment (youngbupark) [inline docs/release-notes/v0.32.0.md:13]: @willtsai @vinayada1 @lakshmimsft Do you have any features you want to highlight here ?

Comment (lakshmimsft) [reply docs/release-notes/v0.32.0.md:13]: adding @willtsai, @Reshrahim

Comment (vinayada1) [reply docs/release-notes/v0.32.0.md:13]: No feature added by me to highlight

**Review by vinayada1 — DISMISSED** [review]

File: docs/release-notes/v0.32.0.md (hunk 1 above)
Comment (Reshrahim) [reply docs/release-notes/v0.32.0.md:13]: we can also highlight some of the community contributions - 

 1. Change open portal hyperlinks by @gpltaylor in https://github.com/radius-project/radius/pull/7229
2. chore: add dependabot configuration by @lechnerc77 in https://github.com/radius-project/radius/pull/7307

File: docs/release-notes/v0.32.0.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+WIP
```
Comment (willtsai) [inline docs/release-notes/v0.32.0.md:15]:
```suggestion
### Support for Terraform modules from private git repositories

Radius Environments can now be configured to pull Terraform Recipe templates from private git repositories. Visit the [private registry how-to guide](https://docs.radapp.io/guides/recipes/howto-private-registry/) to learn more.
```

File: docs/release-notes/v0.32.0.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
```
Comment (Reshrahim) [inline docs/release-notes/v0.32.0.md:16]:
```suggestion
## Community Contributions
-`rad app graph` now supports hyperlinks for Azure resources. Thanks to @gpltaylor 
- Thanks to @lechnerc77 for adding dependabot configuration to all the GitHub features.
```

**Review by Reshrahim — APPROVED** [review]

**Review by vinayada1 — APPROVED** [review]

**Review by kachawla — APPROVED** [review]

**Pull request merged**